Backpack/build.gradle

```diff
@@ -35,6 +35,7 @@ dependencies {
   androidTestImplementation 'androidx.test:rules:1.1.1'
   androidTestImplementation 'com.nhaarman.mockitokotlin2:mockito-kotlin:2.1.0'
   androidTestImplementation 'org.mockito:mockito-android:2.23.4'
+  api 'io.github.inflationx:viewpump:1.0.0'
```

Comment:
does this really need to be `api`? Don't remember where I read it but supposedly `api` should be avoided if possible

Reply:
using `api` will pass down the implementation of the lib to the consumer apps. This is required in our case.